BaseApi.Tests/DynamoDbIntegrationTests.cs

```diff
@@ -0,0 +1,65 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace BaseApi.Tests
+{
+    public class DynamoDbIntegrationTests<TStartup> where TStartup : class
+    {
+        protected HttpClient Client { get; private set; }
+        private DynamoDbMockWebApplicationFactory<TStartup> _factory;
+        protected IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
+        protected List<Action> CleanupActions { get; set; }
+
+        private readonly List<TableDef> _tables = new List<TableDef>
+        {
+            // TODO: Populate the list of table(s) and their key property details here, for example:
```

Comment:
Is this a forgotten comment?

Reply:
No. When running integration tests the service expects the tables to be pre-created, so within the test project startup this list is used to ensure that any required tables exist before any tests are run.